Add udev rules to Tails tutorial (#546)
zkSNACKs/WasabiDoc: docs/using-wasabi/WasabiSetupTails.md

wassabee </dev/null &>/dev/null &
```

## Hardware Wallet tutorial (udev rules)

On Linux you need to create a set of `udev` rules to allow hardware wallet access.

Since Wasabi keeps the configuration files on `$HOME` it's necessary to change persistence in order to keep them.

> So configure Persistance, download Wasabi and unpack it inside `Persistent` folder then

Reviewer: ```suggestion
So configure persistence, download Wasabi and unpack it inside `Persistent` folder then reboot tails with `root` support and launch the following commands from the Terminal:
```

> reboot tails with `root` support and launch the following commands from the Terminal:

Reviewer: ```suggestion
```

Author: ?

Reviewer: This line should be removed.

Author: why?

Reviewer: Because this line should have been written in the same line above it not in a new line, and this is now fixed when you committed my first suggestion, so now this one is redundant.
Read the previous line and you will understand.

```
sudo -i
echo -e "/home/amnesia/.walletwasabi\tsource=walletwasabi" >> /live/persistence/TailsData_unlocked/persistence.conf
mkdir -p /live/persistence/TailsData_unlocked/walletwasabi
chown amnesia:amnesia /live/persistence/TailsData_unlocked/walletwasabi
```

> ### ColdCard Wallet

Reviewer: ```suggestion
### ColdCard devices
```

Reviewer 2: NACK, they are called `ColdCard Wallet`

Reviewer: Then maybe `ColdCard Wallet devices` is better.
How about `Trezor` and `Ledger` and `Keepkey` aren't they also called wallet?

Author: There's just one ColdCard Wallet. Ledger, Trezor and KeepKey have multiple products
